test/Microsoft.AspNetCore.Owin.Tests/OwinExtensionTests.cs

```diff
@@ -10,18 +10,36 @@
 using Microsoft.AspNetCore.Http;
 using Microsoft.Extensions.DependencyInjection;
 using Xunit;
+using System.Net.WebSockets;
```

Comment:
OK, I will update it.

BTW, is there any mandatory code style check in the build system so that this kind of code style inconsistency will cause build error (or at least build warning)?